src/components/Button.tsx

```diff
@@ -23,6 +24,7 @@ export const Button = styled.button`
 export const PrimaryButton = styled(Button)`
   background-color: transparent;
   border-radius: 20px;
+  font-weight: 700;
```

Comment:
i dont think the primary button needs font weight tho 🤔 what page is this for? it shouldnt be added to the global component if it's going to affect the other components

Reply:
I made this change so that the "Add New Class" Primary button would have the bold font weight shown in the design. Is there another way I can update this button? There is a font-weight of 500 for the basic Button styling.

Reply:
<img width="725" alt="Screen Shot 2022-08-24 at 1 28 15 PM" src="https://user-images.githubusercontent.com/40800353/186484640-16225f67-1e9a-44e6-b352-b75462b6d3f9.png">

mm it looks like it's not bolded tho in the design?

Reply:
looks like this if i add font weight of 500
<img width="234" alt="Screen Shot 2022-08-24 at 1 46 03 PM" src="https://user-images.githubusercontent.com/40800353/186487908-2a5cd5da-3539-4fa8-a751-e5875de69b89.png">